packages/site-components/src/VerticalNavigation.tsx

```diff
@@ -17,11 +17,26 @@ export type VerticalNavigationProps = {
 const MenuIcon = ({ open }) => <Icon name={open ? 'chevronDown' : 'chevronRight'} />;
 
 const menuItemStyles = {
-  button: ({ active }) => {
-    return {
-      fontWeight: active ? 'var(--fontWeight-bold)' : 'var(--fontWeight-regular)'
-    };
+  root: {
+    fontSize: '13px'
+  },
+  subMenuContent: {
+    backgroundColor: 'inherit'
+  },
+  button: ({ active }) => ({
+    fontWeight: active ? 'var(--fontWeight-bold)' : 'var(--fontWeight-regular)',
+    '&:hover': {
+      opacity: 0.5,
```

Comment:
temporary fix - would useColorMode() should be used to switch on hover colours between light and dark modes but not working in this file for some reason